### PR TITLE
fmv.rst: Editorial revisions

### PR DESCRIPTION
**Pull request by DougTW**
Editorial changes include:
- added standardized introduction paragraph and description
- rearranged some content into introduction
- added contents directive at the start of the document
- revised some steps to be continuations of other steps

Signed-off-by: DougTW <doug.martin@intel.com>

### CODE REVIEW

File: source/tutorials/fmv.rst [hunk 1]
```diff
@@ -1,36 +1,46 @@
...
-To install the bundles, run the following command in the :file:`$HOME`
-directory:
+#.  To install the bundles, run the following command in the :file:`$HOME`
+    directory:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-   sudo swupd bundle-add desktop-dev
+     sudo swupd bundle-add desktop-dev
```
Comment (mvincerx) [inline source/tutorials/fmv.rst]: Please double-check to make sure `sudo` is left-aligned with `code`, above. Thank you.

File: source/tutorials/fmv.rst [hunk 2]
```diff
@@ -1,36 +1,46 @@
 .. _fmv:
 
-Function multi-versioning
+Function Multi-Versioning
```
Comment (mvincerx) [inline source/tutorials/fmv.rst]: Revisions look great. It's so much cleaner and easy to follow. Just a couple fussy things. 
Please revert header to original. We use `Title case` for our [headers](https://docs.01.org/clearlinux/latest/reference/collaboration/structure-formatting.html). Thanks.

File: source/tutorials/fmv.rst (hunk 1 above)
Comment (DougTW) [reply source/tutorials/fmv.rst]: Done.

File: source/tutorials/fmv.rst (hunk 2 above)
Comment (mvincerx) [reply source/tutorials/fmv.rst]: Stet.

**Comment by mvincerx** [comment]
LGTM. Cheers.

**Pull request merged**